src/core/Link/LinkListItem/LinkListItem.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { forwardRef, ReactElement } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { LinkListItemStyles } from './LinkListItem.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { HtmlLi, HtmlLiProps, HtmlSpan } from '../../../reset';
+import { LinkProps } from '../Link/Link';
+import { ExternalLinkProps } from 'core/Link/ExternalLink/ExternalLink';
+import { RouterLinkProps } from 'core/Link/RouterLink/RouterLink';
+import { Icon } from '../../Icon/Icon';
+import { BaseIconKeys } from 'suomifi-icons';
+
+const baseClassName = 'fi-link-list-item';
+
+const listLinkClassNames = {
+  icon: `${baseClassName}_icon`,
+};
+
+export interface LinkListItemProps extends Omit<HtmlLiProps, 'className'> {
+  /** Ref  is passed to the anchor element. Alternative to React `ref` attribute. */
+  forwardedRef?: React.Ref<HTMLLIElement>;
+  /**
```

Comment:
Also these could be oneliners like first prop

Reply:
Changed these.